kuksa-sdk/src/main/kotlin/org/eclipse/kuksa/DataBrokerConnection.kt

```diff
@@ -124,6 +191,46 @@ class DataBrokerConnection internal constructor(
         }
     }
 
+    /**
+     * Retrieves the [VssSpecification] and returns it to the corresponding Callback. The retrieved [VssSpecification]
+     * is of the same type as the inputted. All underlying heirs are changed to reflect the data broker state.
+     *
+     * @param specification to retrieve
+     * @param fields to retrieve. The default value is a list with a single [Types.Field.FIELD_VALUE] entry.
```

Comment:
I only added dots when there were at least two sentences. It would look weird to have the first sentence with a dot and the following not? Any ideas?

Reply:
Maybe we should follow this one? https://kotlinlang.org/docs/coding-conventions.html#avoid-redundant-constructs

Reply:
> https://kotlinlang.org/docs/coding-conventions.html#avoid-redundant-constructs

Don't really see how the three rules could be applied here?
 
> Any ideas?

Well, we have a lot of options here:

1. Reformulate the sentence so these are not two sentences
2. Always end with a dot, independent on if it's a half sentence or not
3. Keep it flexible as is
4. Improve overall comment quality - don't allow half sentences for param, return, throws and further and therefore always end with a "."

Reply:
The anchor point was wrong from the link (gotta scroll up a bit). We agreed to follow the "recommended" way here to omit the @param whenever possible to reduce redundant sentences.